src/axom/quest/SignedDistance.hpp

```diff
@@ -62,8 +74,25 @@ struct UcdMeshData
    *
    * \return pointer to the node IDs in the underlying cell node array.
    */
-  AXOM_HOST_DEVICE const IndexType* getCellNodeIDs(IndexType cellId,
-                                                   int& nnodes) const;
+  AXOM_HOST_DEVICE const IndexType* getCellNodeIDs(IndexType cellId, int& nnodes) const
```

Comment:
Similarly here, does this need to be marked `inline` ?

Reply:
My guess is that the compiler would not inline this due to the complexity (conditional logic).